pkg/egressgateway/manager.go

```diff
@@ -611,7 +611,7 @@ func (manager *Manager) relaxRPFilter() error {
 		if _, ok := ifSet[ifaceName]; !ok {
 			ifSet[ifaceName] = struct{}{}
 			sysSettings = append(sysSettings, tables.Sysctl{
-				Name:      fmt.Sprintf("net.ipv4.conf.%s.rp_filter", ifaceName),
+				Name:      fmt.Sprintf("net.ipv4.conf.%s.rp_filter", strings.ReplaceAll(ifaceName, ".", "/")),
```

Comment:
This doesn't actually fix the issue, when the sysctl package converts the remaining dots you end up with the same result.